bootcamp-hub/values.yaml

```diff
@@ -0,0 +1,54 @@
+jupyterhub:
+  hub:
+    baseUrl: /bootcamp-hub/
+    annotations:
+      prometheus.io/scrape: "true"
+      # this needs to start with the value of `hub.baseUrl`
+      prometheus.io/path: /bootcamp-hub/hub/metrics
+    extraConfig:
+      auth: |
+        c.JupyterHub.authenticator_class = 'hashauthenticator.HashAuthenticator'
+      admin: |
+        c.Authenticator.admin_users = {'leah-admin', 'tim-admin'}
+        c.JupyterHub.admin_access = True
+    image:
+      # tag will be set by travis on deployment
+      name: earthlabhubops/ea-k8s-hub-wshub
```

Comment:
What image do you want to use? If you want to have a self-built image then I'd suggest we stick to the naming convention of `earthlabhubops/ea-k8s-hub-<hubname>` with the source of that image in `hub-images/<hubname>`. This is what the travis setup assumes and let's it find out if there is a custom image for a hub, if it needs to rebuild it, what the tag is, etc. We can make it all more auto discovery and super cross hub flexible however getting it rock solid is a lot of work. So right now I would prefer to lean on humans and conventions instead of investing in that level of functionality. What do you think?

(I am adding to the docs https://github.com/earthlab/hub-ops/pull/79 to write down all these conventions)

Reply:
Note: this sets the image used by the hub itself. Here you can customise things like adding a exotic authenticator. Hash authenticator is such an exotic authenticator. Trying to think of other reasons you'd need this: none. So mostly having a custom hub image is for when you need a custom authenticator.

Reply:
ok so this triggers the build of a new docker container specifically for this "pool" or hub. 

how about this: 
`      name: earthlabhubops/ea-k8s-hub-bootcamp-hub` 
now i'm realizing that by having a dash - the readability of this line is a bit lower hub-bootcamp-hub... vs hub-bootcamphub. i just like dashes in names for readability in general. 

thoughts?

Reply:
Yes? I don't think things will break if there is an extra dash. You choose.